apps/events/models.py

```diff
@@ -858,6 +841,9 @@ def delete(self):
             # Do nothing
             False
 
+        if not self.is_on_waitlist():
+            self.event.notify_waiting_list()
```

Comment:
I'm reading this as "if noone is on waitlist; notify waitlist". Is that correct? 🤔

Reply:
No, it should be read as `if not attendee is on waitlist`.
Some context is lost in the diff as this is really a method on the Attendee class. 
More specifically Attendee.delete.

Reply:
Ah, right, that's where the confusion arised.